src/modules/finance/index.ts

```diff
@@ -516,6 +522,19 @@ export class FinanceModule extends ModuleBase {
     return symbol;
   }
 
+  /**
+   * Returns a random currency numeric code.
+   * (The ISO 4217 numerical code for the currency (e.g. `US Dollar` -> `'840'` ))
```

Comment:
Are the parenthesis required here? Additionally, since this function returns a random code, it should be "a currency" not "the curreny":


```suggestion
   * The ISO 4217 numerical code for a currency (e.g. `US Dollar` -> `'840'` )
```